dev-db/mariadb/mariadb-11.4.3.ebuild

```diff
@@ -9,11 +9,15 @@ JAVA_PKG_OPT_USE="jdbc"
 inherit systemd flag-o-matic prefix toolchain-funcs \
 	multiprocessing java-pkg-opt-2 cmake
 
+DESCRIPTION="An enhanced, drop-in replacement for MySQL"
 HOMEPAGE="https://mariadb.org/"
 SRC_URI="mirror://mariadb/${PN}-${PV}/source/${P}.tar.gz
-	https://github.com/hydrapolic/gentoo-dist/raw/main/mariadb/mariadb-11.4.2-patches-01.tar.xz"
+	https://github.com/hydrapolic/gentoo-dist/raw/main/mariadb/${PN}-11.4.3-patches-01.tar.xz"
```

Comment:
Why do we need the `0005-disable-ed25519-for-server.patch`? I miss this information in the patch.
```diff
# partially revert https://github.com/mariadb-corporation/mariadb-connector-c/commit/dddcf400af9a693fdbed4e692d71bf98b79b7aa1
diff --git a/libmariadb/plugins/auth/CMakeLists.txt b/libmariadb/plugins/auth/CMakeLists.txt
index 3d2ede14..c55bb178 100644
--- a/libmariadb/plugins/auth/CMakeLists.txt
+++ b/libmariadb/plugins/auth/CMakeLists.txt
@@ -69,11 +69,6 @@ IF(CRYPTO_PLUGIN)
                 INCLUDES ${REF10_DIR}
                 LIBRARIES ${CRYPT_LIBS}
                 COMPILE_OPTIONS -DMYSQL_CLIENT=1)
-  SET_PROPERTY(TARGET client_ed25519 PROPERTY C_STANDARD 11)
-
-  IF(HAVE_C_THREAD_LOCAL)
-    TARGET_COMPILE_DEFINITIONS(client_ed25519 PRIVATE -DHAVE_THREAD_LOCAL)
-  ENDIF()

   IF(MSVC)
      # Silence conversion (integer truncation) warnings from reference code
```

Reply:
The configuration stage failed with:
```
...
-- Looking for floor
-- Looking for floor - not found
-- Looking for pthread_getspecific
-- Looking for pthread_getspecific - found
-- Looking for gethostbyname_r
-- Looking for gethostbyname_r - found
-- Looking for setsockopt
-- Looking for setsockopt - found
-- TLS library/version: OpenSSL 3.3.2
-- SYSTEM_LIBS /usr/lib64/libz.so;dl;m;dl;m;/usr/lib64/libssl.so;/usr/lib64/libcrypto.so
CMake Error at libmariadb/plugins/auth/CMakeLists.txt:72 (SET_PROPERTY):
  SET_PROPERTY could not find TARGET client_ed25519.  Perhaps it has not yet
  been created.
Call Stack (most recent call first):
  libmariadb/plugins/CMakeLists.txt:7 (INCLUDE)
  libmariadb/CMakeLists.txt:435 (INCLUDE)


CMake Error at libmariadb/plugins/auth/CMakeLists.txt:75 (TARGET_COMPILE_DEFINITIONS):
  Cannot specify compile definitions for target "client_ed25519" which is not
  built by this project.
Call Stack (most recent call first):
  libmariadb/plugins/CMakeLists.txt:7 (INCLUDE)
  libmariadb/CMakeLists.txt:435 (INCLUDE)
```

Reply:
This might be caused by a wrongly updated patch, see the diff between original and fixed patch:
```diff
 diff --git a/libmariadb/libmariadb/CMakeLists.txt b/libmariadb/libmariadb/CMakeLists.txt
-index 852be8dc..0373eb4c 100644
+index 1985b44f..5b77ec40 100644
 --- a/libmariadb/libmariadb/CMakeLists.txt
 +++ b/libmariadb/libmariadb/CMakeLists.txt
-@@ -447,8 +447,10 @@ IF(WIN32)
+@@ -449,8 +449,10 @@ IF(WIN32)
                     "FILE_DESCRIPTION:Dynamic lib for client/server communication")
  ENDIF()

 -ADD_LIBRARY(mariadbclient STATIC  ${MARIADB_OBJECTS} ${EMPTY_FILE})
--TARGET_LINK_LIBRARIES(mariadbclient ${SYSTEM_LIBS} ${CRYPTO_LIBS})
+-TARGET_LINK_LIBRARIES(mariadbclient ${SYSTEM_LIBS} ${CRYPT_LIBS})
 +IF(ENABLE_STATIC_LIBS OR NOT WITHOUT_TOOLS)
 + ADD_LIBRARY(mariadbclient STATIC  ${MARIADB_OBJECTS} ${EMPTY_FILE})
-+ TARGET_LINK_LIBRARIES(mariadbclient ${SYSTEM_LIBS} ${CRYPTO_LIBS})
++ TARGET_LINK_LIBRARIES(mariadbclient ${SYSTEM_LIBS} ${CRYPT_LIBS})
 +ENDIF(ENABLE_STATIC_LIBS OR NOT WITHOUT_TOOLS)

  IF(UNIX)
    ADD_LIBRARY(libmariadb SHARED ${libmariadb_RC} ${MARIADB_OBJECTS} ${EMPTY_FILE})
```
I am running tests for `11.4.4` locally.

Reply:
Oops, sorry for that!